osu.Game/Overlays/Settings/SettingsItem.cs

```diff
@@ -96,7 +98,23 @@ public virtual Bindable<T> Current
             set => controlWithCurrent.Current = value;
         }
 
-        public virtual IEnumerable<string> FilterTerms => Keywords == null ? new[] { LabelText.ToString() } : new List<string>(Keywords) { LabelText.ToString() }.ToArray();
+        public virtual IEnumerable<string> FilterTerms
+        {
+            get
+            {
+                var keywords = new List<string>(Keywords ?? Array.Empty<string>())
+                {
+                    LabelText.ToString()
+                };
+
+                if (GetClassicDefault != null)
+                {
+                    keywords.Add(CLASSIC_DEFAULT_SEARCH_TERM);
+                }
```

Comment:
This looks a touch... bolted onto the search feature, and as a consequence, if you type `has` in the actual settings sidebar, it triggers this filter and shows only settings that have classic defaults. Not sure whether that was intended to be a feature, or just a byproduct of the bolt-on.

Reply:
It can be done a different way, ie. using a custom search container that does `IFilterable` and matches on `ClassicDefault!=null`. But I felt that this was a much simpler implementation and had few side effects. I do think it's a feature that you can search this at the main settings, and it might be something we use in the future, who knows.

Reply:
I'll look away for the time being and see how the user feedback from this goes.